### ISSUE TITLE
live reloading stopped working in React Native 0.57.2

### ISSUE THREAD

**Issue by xfyre** [bhauman/lein-figwheel#716]
Live reloading doesn't seem to work any longer after upgrade to RN 0.57.2.

From Chrome console everything seems to be working:

```
FigWheel: Attempting to load http://localhost:8081/target/ios/goog/../dcnet_app/screens/home.js
utils.cljs?rel=1540401243619:63 FigWheel: Successfully loaded http://localhost:8081/target/ios/goog/../dcnet_app/screens/home.js
utils.cljs?rel=1540401243619:63 FigWheel: Attempting to load http://localhost:8081/target/ios/goog/../dcnet_app/appnav.js
utils.cljs?rel=1540401243619:63 FigWheel: Successfully loaded http://localhost:8081/target/ios/goog/../dcnet_app/appnav.js
utils.cljs?rel=1540401243619:63 FigWheel: Attempting to load http://localhost:8081/target/ios/goog/../dcnet_app/ios/core.js
blob:http://localhost:8081/906ddafd-c5c3-41bc-bfd3-e6cd45e2ab4f:146405 Requiring: react-native
utils.cljs?rel=1540401243619:63 FigWheel: Successfully loaded http://localhost:8081/target/ios/goog/../dcnet_app/ios/core.js
utils.cljs?rel=1540401243619:63 Figwheel: Not trying to load file 
utils.cljs?rel=1540401243619:72 ("../dcnet_app/screens/home.js" "../dcnet_app/appnav.js" "../dcnet_app/ios/core.js")
utils.cljs?rel=1540401243619:72 figwheel-no-load meta-data: ("../env/ios/main.js")
```

But the final refresh in the simulator isn't happening.

**Comment by danielcompton** [comment]
@xfyre can you give more information on your setup? Your target (iOS simulator?), OS version, version of Figwheel, ClojureScript version, e.t.c.

Also, what was the last version where this worked successfully, did you upgrade anything else while upgrading React Native?

**Comment by xfyre** [comment]
@danielcompton sure:
- Target is iOS simulator, iOS 12
- MacOS 10.13.6 (17G65)
- lein-figwheel "0.5.17", figwheel-sidecar "0.5.17", com.cemerick/piggieback "0.2.2", re-frame "0.10.5", reagent "0.8.1", org.clojure/clojurescript "1.10.238", org.clojure/clojure "1.9.0"
- last working version was the one installed with the latest stable re-natal (0.55.4)

After upgrading RN I added NativeBase, but I don't think I changed anything else. Yesterday I've also upgraded re-natal from [this PR](https://github.com/drapanjanas/re-natal/pull/192), hoping it would help.

I can try to track it down if you tell me what to look for. Turning on inspection in emulator shows that reloader element is there: https://monosnap.com/direct/T71Bori2dQmDbo8udbNuMY4OIBxNcx

But the actual refresh somehow never happens.
